starter-code/styles/modules.css

```diff
@@ -55,3 +59,36 @@
          - Main nav should be all on one row, with padding spaced better for a larger screen.
          - Just, ya know, try to make it look good! */
 /* @media only screen... */
+@media all and (min-width: 700px) {
+    /*.icon-menu {
+      display: none;
```

Comment:
Always ensure to remove, code that isn't in use before pushing up. Just a good habit.